Mathlib/Topology/ContinuousMap/Algebra.lean

```diff
@@ -372,25 +372,25 @@ instance [CommGroup β] [TopologicalGroup β] : TopologicalGroup C(α, β) where
       uniformContinuous_inv.comp_tendstoUniformlyOn
         (tendsto_iff_forall_compact_tendstoUniformlyOn.mp Filter.tendsto_id K hK)
 
--- TODO: rewrite the next three lemmas for products and deduce sum case via `to_additive`, once
--- definition of `tprod` is in place
-/-- If `α` is locally compact, and an infinite sum of functions in `C(α, β)`
-converges to `g` (for the compact-open topology), then the pointwise sum converges to `g x` for
-all `x ∈ α`. -/
-theorem hasSum_apply {γ : Type*} [AddCommMonoid β] [ContinuousAdd β]
-    {f : γ → C(α, β)} {g : C(α, β)} (hf : HasSum f g) (x : α) :
-    HasSum (fun i : γ => f i x) (g x) := by
-  let ev : C(α, β) →+ β := (Pi.evalAddMonoidHom _ x).comp coeFnAddMonoidHom
-  exact hf.map ev (ContinuousMap.continuous_eval_const x)
-
-theorem summable_apply [AddCommMonoid β] [ContinuousAdd β] {γ : Type*} {f : γ → C(α, β)}
-    (hf : Summable f) (x : α) : Summable fun i : γ => f i x :=
-  (hasSum_apply hf.hasSum x).summable
-
-theorem tsum_apply [T2Space β] [AddCommMonoid β] [ContinuousAdd β] {γ : Type*} {f : γ → C(α, β)}
-    (hf : Summable f) (x : α) :
-    ∑' i : γ, f i x = (∑' i : γ, f i) x :=
-  (hasSum_apply hf.hasSum x).tsum_eq
+/-- If an infinite sum of functions in `C(α, β)` converges to `g` (for the compact-open topology),
+then the pointwise sum converges to `g x` for all `x ∈ α`. -/
+@[to_additive]
```

Comment:
This is the additive docstring on the multiplicative statement.

Reply:
:facepalm: